src/main/java/com/minecolonies/coremod/commands/ActionMenuType.java

```diff
@@ -0,0 +1,300 @@
+            CitizenInfoCommand.class,
+            new ActionArgument("colony", ActionArgumentType.Colony, ActionArgumentType.Is.Required,
+                    new ActionArgument("citizen", ActionArgumentType.Citizen, ActionArgumentType.Is.Required))
+        )),
+
+    ShowColonyInfo(new ActionMenu(
+            "ShowColonyInfo",
+            "info",
+            ForgePermissionNodes.ShowColonyInfo,
+            ShowColonyInfoCommand.class,
+            // TODO: need OR() ActionArgumentType
+            new ActionArgument("player", ActionArgumentType.Player, ActionArgumentType.Is.Optional),
+            new ActionArgument("colony", ActionArgumentType.Colony, ActionArgumentType.Is.Optional)
+            )),
+    DeleteColony(new ActionMenu(
+            "DeleteColony",
+            "delete",
+            ForgePermissionNodes.DeleteColony,
+            DeleteColonyCommand.class,
+            new ActionArgument("colony", ActionArgumentType.Colony, ActionArgumentType.Is.Required),
+            new ActionArgument("canDestroy", ActionArgumentType.BOOLEAN, ActionArgumentType.Is.Optional),
+            new ActionArgument("confirmDelete", ActionArgumentType.BOOLEAN, ActionArgumentType.Is.Optional)
+            )),
+    DisableBarbarianSpawns(new ActionMenu(
+            "DisableBarbarianSpawns",
+            "barbarians",
+            ForgePermissionNodes.DisableBarbarianSpawns,
+            DisableBarbarianSpawnsCommand.class,
+            new ActionArgument("colony", ActionArgumentType.Colony, ActionArgumentType.Is.Required),
+            new ActionArgument("disableSpawns", ActionArgumentType.BOOLEAN, ActionArgumentType.Is.Required)
+            )),
+    AddOfficer(new ActionMenu(
+            "AddOfficer",
+            "addOfficer",
+            ForgePermissionNodes.AddOfficer,
+            AddOfficerCommand.class,
+            new ActionArgument("colony", ActionArgumentType.Colony, ActionArgumentType.Is.Required),
+            new ActionArgument("player", ActionArgumentType.Player, ActionArgumentType.Is.Required)
+            )),
+    RefreshColony(new ActionMenu(
+            "RefreshColony",
+            "refresh",
+            ForgePermissionNodes.RefreshColony,
+            RefreshColonyCommand.class,
+            // TODO: need OR() ActionArgumentType
+            new ActionArgument("player", ActionArgumentType.Player, ActionArgumentType.Is.Optional),
+            new ActionArgument("colony", ActionArgumentType.Colony, ActionArgumentType.Is.Optional)
+            )),
+    ChangeColonyOwner(new ActionMenu(
+            "Ownership Change",
+            "ownerchange",
+            ForgePermissionNodes.ChangeColonyOwner,
+            ChangeColonyOwnerCommand.class,
+            new ActionArgument("colony", ActionArgumentType.Colony, ActionArgumentType.Is.Required),
+            new ActionArgument("player", ActionArgumentType.Player, ActionArgumentType.Is.Required)
+            )),
+    ColonyTeleport(new ActionMenu(
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00115)